src/garage/experiment/local_runner.py

```diff
@@ -54,13 +118,29 @@ def __init__(self, snapshot_config, max_cpus=1):
                                         snapshot_config.snapshot_gap)
 
         if max_cpus > 1:
+            # pylint: disable=import-outside-toplevel
             from garage.sampler import singleton_pool
             singleton_pool.initialize(max_cpus)
-        self.has_setup = False
-        self.plot = False
+        self._has_setup = False
+        self._plot = False
 
         self._setup_args = None
-        self.train_args = None
+        self._train_args = None
+        self._stats = ExperimentStats(total_itr=0,
+                                      total_env_steps=0,
+                                      total_epoch=0,
+                                      last_path=None)
+
+        self._algo = None
+        self._env = None
+        self._policy = None
+        self._sampler = None
+        self._plotter = None
+
+        self._start_time = None
+        self._itr_start_time = None
+        self.step_itr = None
```

Comment:
please make these _private

Reply:
This is a variable that gets updated in algorithms.